README.md

```diff
@@ -131,6 +131,20 @@ Downloads the RHCOS initrd with the ignition for the specified image appended.
 
 - `Authorization`: this header is passed directly through to assisted service requests to handle RHSSO authentication
 
+### `GET /images/{image_id}/initrd-addrsize`
```

Comment:
I would like to emphasize how specific this is, so let's have it be part of the name

```suggestion
### `GET /images/{image_id}/s390x-initrd-addrsize`
```

Reply:
Or are there other IBM architectures which this might be relevant to in the future?

Reply:
No yet ... so for the time being s390x only.

Reply:
Done